src/metabase/api/setup.clj

```diff
@@ -86,4 +88,79 @@
       (catch Throwable e
         (response-invalid :general (.getMessage e))))))
 
+
+(defn- admin-checklist []
+  (let [has-dbs?           (db/exists? Database)
+        has-dashboards?    (db/exists? 'Dashboard)
+        has-pulses?        (db/exists? 'Pulse)
+        has-labels?        (db/exists? 'Label)
+        has-hidden-tables? (db/exists? 'Table, :visibility_type [:not= nil])
+        has-metrics?       (db/exists? 'Metric)
+        has-segments?      (db/exists? 'Segment)
+        num-tables         (db/select-one-count 'Table)
+        num-cards          (db/select-one-count 'Card)
+        num-users          (db/select-one-count 'User)]
+    [{:name        "Add a database"
```

Comment:
@camsaul Would it be possible to break this into "sections" like the fixture JSON in https://github.com/metabase/metabase/pull/2638/files#diff-bddb7038f5a7480939655d41622e6ed2R4? It'd be possible to break the list into themes client side but seems nicer to just have the API response be the right shape from the get go.